create/how_to/aws.md

```diff
@@ -0,0 +1,181 @@
+
+# Deploy a MeiliSearch Instance on AWS
+
+Using our **MeiliSearch [AMI](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AMIs.html)**, MeiliSearch can be deployed on AWS in just a few minutes.
+The following `how-to guide` will walk you through every step to deploy MeiliSearch in an AWS EC2 instance.
+
+**Table of Contents**:
+
+[[toc]]
+
+## Part 1: Deploy an out-of-the-box MeiliSearch instance
+
+### 1. Launch an instance from the AWS console
+
+After logging into your [AWS Console](https://aws.amazon.com/console), navigate to the **Compute** service. Then go to **EC2**, and finally open your **Instances** console.
+
+![Launch an instance from AWS console](/aws/01.launch-instances.png)
+
+In the top-right corner, click on the **Launch instances** button to start the process of configuring your MeiliSearch instance.
+
+### 2. Select 'MeiliSearch' AMI from 'Community AMIs'
+
+You will now select which AMI or system Image to use to run your instance. Type **"meilisearch"** in the searchbar and select the **Community AMIs** tab on the left sidebar.
+
+![Choose MeiliSearch AMI from Community AMIs](/aws/02.select-ami.png)
+
+Click on **Select** (right side of the screen) to confirm your choice.
+
+### 3. Size and specs
+
+Select the specifications of the server you want MeiliSearch to run on.
+
+![Size and specs](/aws/03.size-and-specs.png)
+
+> We recommend prioritizing memory allocation for better MeiliSearch performance.
```

Comment:
I can't find where we mention this in the doc right now, but we should point to the section where we elaborate on minimum specifications or explanation on why memory is a possible issue. It will give more context for people that want it.

Reply:
Totally agree. It's [still a PR](https://github.com/meilisearch/documentation/pull/772)

Reply:
oh, once @qdequele approve the PR, we can add the link. No need to wait for it for this PR, but be sure to remember to add it once it's merged.